indigo_drivers/mount_ioptron/README.md

```diff
@@ -20,11 +20,15 @@ indigo_server indigo_mount_ioptron
 
 ## Comments
 
-Use URL in form ieq://host:port to connect to the mount over network (default port is 4030).
+Use URL in form ieq://host:port to connect to the mount over network. The default port can be found through the hand controller going to Settings < Wi-Fi Options < Wireless Status
+
+**EXAMPLE OF SETTING IN INDIGO**
+
+![Alt text](ioptronsettings.jpg)
 
 ## Status: Stable
 
 Driver is tested with:
-* CEM60-EC
-* iEQ 45 Pro
-
+* CEM60-EC | Default Port 4030
```

Comment:
I do not think the default port should be listed in Driver status section. It should probably be moved in the Comments.